### PR TITLE
Implement subtasks restart cost estimation RPC

### PR DESCRIPTION
**Pull request by kmazurek**
- Added the RPC endpoint `comp.task.subtasks.estimated.cost`.
- Added `disable_concent` flag to the `comp.task.restart` endpoint.
- Fixed funds checking when performing a partial restart of a task (including previously failed subtasks).

### CODE REVIEW

File: golem/task/rpc.py [hunk 1]
```diff
@@ -605,12 +606,20 @@ def restart_subtasks(
         :return: In case of any errors, returns the representation of the error
         (either a string or a dict). Otherwise, returns None.
         """
-        try:
-            task = self.task_manager.tasks[task_id]
-        except KeyError:
-            err_msg = f'Task not found: {task_id!r}'
-            logger.error(err_msg)
-            return err_msg
+        if not subtask_ids:
```
Comment (shadeofblue) [inline golem/task/rpc.py]: @zakaprov actually, providing an empty array _should_ be legal here... in which case, we want to restart the task by repeating only the failed subtasks ... and only if we _do_ provide an array, we want to _also_ include the specified list of subtasks to restart...

**Comment by shadeofblue** [comment]
@zakaprov -> generally, please run `node_integration_tests/tests/test_golem.py` on the branch to make sure

File: golem/task/rpc.py (hunk 1 above)
Comment (kmazurek) [reply golem/task/rpc.py]: You're right, let me revert this.

**Comment by kmazurek** [comment]
@shadeofblue I ran the integration tests against this branch locally with all tests passing. Tried the same on Buildbot as well ([here](https://buildbot.golem.network/buildbot/#/builders/15/builds/717)), with three of the tests failing for reasons unrelated to my changes (geth nodes being out of sync).

**Review by shadeofblue — APPROVED** [review]
lgtm

**Pull request merged**